mitmproxy/script/concurrent.py

```diff
@@ -11,7 +11,7 @@ class ScriptThread(basethread.BaseThread):
     name = "ScriptThread"
 
 
-def concurrent(fn):
+def concurrent(fn):  # pragma: no coverage windows
```

Comment:
I wasn't aware that it SHOULD work on windows. Why do the existing tests not cover it then?